installer/install.sh

```diff
@@ -79,40 +75,24 @@ argocd app sync science-platform \
   --port-forward-namespace argocd
 
 echo "Syncing critical early applications"
-if [ $(yq -r .ingress_nginx.enabled ../science-platform/values-$ENVIRONMENT.yaml) == "true" ];
-then
-  echo "Syncing ingress-nginx..."
-  argocd app sync ingress-nginx \
-    --port-forward \
-    --port-forward-namespace argocd
-fi
+argocd app sync ingress-nginx \
+  --port-forward \
+  --port-forward-namespace argocd
```

Comment:
This change and the rest of the file all seem erroneous as well. Not sure if this was reverting later changes or trying to get rid of yq, but in any case these changes aren't correct.